script/pktgen_dtap0.sh

```diff
@@ -14,10 +14,12 @@ function add_dtap() {
 }
 
 function set_dtap() {
-    echo "min_pkt_size 64" > /proc/net/pktgen/dtap0
-    echo "max_pkt_size 9000" > /proc/net/pktgen/dtap0
-    echo "count 20" > /proc/net/pktgen/dtap0
+    echo "min_pkt_size 90000" > /proc/net/pktgen/dtap0
+    echo "max_pkt_size 90000" > /proc/net/pktgen/dtap0
```

Comment:
I've not handled the packet segmentation, therefore, it's better to have smaller packet size to avoid unexpected behavior.